plugins/jenkins-backend/src/service/jenkinsApi.ts

```diff
@@ -99,8 +99,16 @@ export class JenkinsApiImpl {
         tree: JenkinsApiImpl.jobsTreeSpec.replace(/\s/g, ''),
       });
 
-      // TODO: support this being a project itself.
-      for (const jobDetails of folder.jobs) {
+      const isStandaloneProject = !project.jobs;
```

Comment:
Just double checking here - does `project.jobs` turn out to be null? Should we be checking `!project.jobs?.length` instead or so?

Reply:
jobs is undefined on project when it is a Pipeline (standalone) project. I don't think it can be a standalone project even if the length is 0.